python/ray/train/tests/test_mosaic_trainer.py

```diff
@@ -88,22 +86,6 @@ def trainer_init_per_worker(config):
 trainer_init_per_worker.__test__ = False
 
 
-def test_mosaic_cifar10(ray_start_4_cpus):
-    from ray.train.examples.mosaic_cifar10_example import train_mosaic_cifar10
-
-    result = train_mosaic_cifar10(max_duration="5ep").metrics_dataframe
-
-    # check the max epoch value
-    assert result["epoch"][result.index[-1]] == 4
-
```

Comment:
no need for this because we're already running the example. also no need to test that mosaic is working correctly.